doc/userguide/src/ExtendingRobotFramework/CreatingTestLibraries.rst

```diff
@@ -157,21 +139,17 @@ the libraries used in the above example:
       def send_message(self, message):
           self._conn.send(message)
 
-.. sourcecode:: java
+.. sourcecode:: python
 
-   public class AnotherLib {
-       private String setting = null;
+   class AnotherLib:
+       setting = None
 
-       public AnotherLib(String setting) {
-           setting = setting;
-       }
+       def __init__(self, setting):
+           self.setting = setting
 
-       public void doSomething() {
-           if setting.equals("42") {
+       def do_something(self):
+           if setting.equals("42"):
```

Comment:
Should be `self.setting == "42"` instead of `setting.equals("42")`. 

Comparing something to "42" is pretty dummy. A bit more realistic example would be renaming `setting` to `environment` and use e.g.
```python
if self.environment == 'test':
    # do something in test environment
else:
    # do something in other environments
```